l10n-ms/src/main/java/com/example/l10nadmin/domain/L10n.java

```diff
@@ -15,28 +12,59 @@
  * @version 2021.04.02
  */
 @Entity
-@Data	// = @ToString, @EqualsAndHashCode, @Getter, @Setter and @RequiredArgsConstructor
+@Table(name = "L10n")
```

Comment:
Is there a reason to explicitly give the name of the table? The default table name is the name of the class.  
If not it's preferable to avoid using boilerplate annotations, so delete this.

The same holds true for `@Column` annotations